HousingManagementSystemApi.Tests/UseCasesTests/RetrieveAddressesUseCaseTests.cs

```diff
@@ -64,6 +79,9 @@ public async Task GivenAPostcode_WhenAnAddressExistsWithAnIneligibleAssetType_Th
             retrieveAssetGateway.Setup(x => x.RetrieveAsset("assetId"))
                 .ReturnsAsync(new AssetResponseObject { AssetType = AssetType.Concierge });
 
+            tenureGateway.Setup(x => x.RetrieveTenureType("Id"))
```

Comment:
Given the test assertion is based on addresses being filtered out by asset type, is the setup of tenure gateway required?

Reply:
RetrieveAddressesUseCase.Execute will also filter on tenure type, which is arguably doing more than it should but I think we agreed this beforehand.